### DISCUSSION TITLE
Alternative web browser engine ?

### DISCUSSION THREAD

**Discussion by Tuxino88** [eneshecan/whatsapp-for-linux#82] [Ideas]
whatsapp will enable calls and video calls in the future. However she is not interested in webkit. Wouldn't it be worth considering a different web browser engine? (gecko/blink) maybe not electron, too heavy and cumbersome.

**Comment by xeco23** [comment]
> whatsapp will enable calls and video calls in the future. However she is not interested in webkit. Wouldn't it be worth considering a different web browser engine? (gecko/blink) maybe not electron, too heavy and cumbersome.

Hi, yeah I think it would be nice to switch to another web engine which whatsapp supports for media capture but I don't see any better alternatives that support C++ for now. However, I haven't put a lot of effort in researching those. Will check your suggestions.

**Reply by Tuxino88** [reply]
I'm sorry I'm not a programmer, I would have gladly helped you. :S Doesn't Gecko allow it?

**Reply by xeco23** [reply]
I think it does but embedding that into the gtk application is another problem since it's not a gtk object. With webkit it's easy because the interface is already a GObject.
